main/src/library/Fixpoint/Ram/RamTerm.flix

```diff
@@ -0,0 +1,45 @@
+/*
+ * Copyright 2021 Benjamin Dahse
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Fixpoint/Ram {
+    pub enum RamTerm[v] {
+        case Lit(v)
+        case Attribute(RowVar, Int32)
+        case Lat(RowVar)
```

Comment:
I would like to get rid of `Lat` entirely, but that's supposed to be a performance improvement so that should wait. For now I'll add documentation for both of them.

Reply:
Fixed.

Reply:
OK, lets rename Attribute/Lat to something more meaningful.